Documentation/ApiOverview/Events/Events/FrontendLogin/BeforeRedirectEvent.rst

```diff
@@ -7,14 +7,8 @@
 BeforeRedirectEvent
 ===================
 
-.. versionadded:: 10.4
-   This event replaces the :php:`$GLOBALS['TYPO3_CONF_VARS']['EXTCONF']['felogin']['beforeRedirect']`
```

Comment:
Mentioning the hook eases the process of migrating from an older version (like v9). So I opt for not removing it. And also on other places where you removed it.

Reply:
Well, for how long would you keep them? Someone googeling for the hooks name would still find it in the v10 Doku...

Reply:
For v11, remove for v12 is okay for me. Would hold it for two versions.